numba_dppy/spirv_generator.py

```diff
@@ -105,7 +105,7 @@ def generate(self, ipath, opath, llvm_spirv_args):
             llvm_spirv_call_args = ["llvm-spirv"]
         if llvm_spirv_args is not None:
             llvm_spirv_call_args += llvm_spirv_args
-        llvm_spirv_call_args += ["-o", opath, ipath + ".bc"]
+        llvm_spirv_call_args += [*llvm_spirv_flags, "-o", opath, ipath + ".bc"]
```

Comment:
I have reverted this change and test PASSED. Why it could be?

Reply:
Because (as I said earlier) this flag only fixes error in spirv-val mode.

Reply:
> spirv-val mode

Could you enable this mode in test?

Reply:
Yes, I can, but checking spirv (spirv-val mode) is blocked by a set of problems. This flag fixes only one problem connected to appropriate Debug name in spirv